test/model/content_test.dart

```diff
@@ -875,6 +893,8 @@ void main() {
       ]]),
     ]);
 
+  testParseExample(ContentExample.thematicBreaks);
```

Comment:
I think a better place to put this is just below the `'parse <br> in block context'` test. That seems consistent with the choice to define `ThematicBreakNode` just after `LineBreakNode`.